app/controllers/photos_controller.rb

```diff
@@ -8,19 +16,23 @@ def index
 
   # GET /photos/1 or /photos/1.json
   def show
+    authorize @photo
   end
 
   # GET /photos/new
   def new
     @photo = Photo.new
+    authorize @photo
   end
 
   # GET /photos/1/edit
   def edit
+    authorize @photo
   end
 
   # POST /photos or /photos.json
   def create
+    authorize @photo
```

Comment:
move `authorize @photo` to line 38 to prevent the error message when creating a post.